extensions/wikia/InsightsV2/InsightsHelper.php

```diff
@@ -198,7 +198,9 @@ public function prepareInsightsList( $limit = 0 ) {
 			$insightsList[$key] = [
 				'subtitle' => self::INSIGHT_SUBTITLE_MSG_PREFIX . $key,
 				'description' => self::INSIGHT_DESCRIPTION_MSG_PREFIX . $key,
-				'count' => $this->prepareCountDisplay( $insightsCountService->getCount( $key ) ),
+				'count' => $class::INSIGHT_USAGE == InsightsModel::INSIGHTS_USAGE_ACTIONABLE
```

Comment:
And as for moving INSIGHT_USAGE to config, we could retrieve it's state from config here.